config/config.go

```diff
@@ -34,6 +35,7 @@ func Get() (*Config, error) {
 		GracefulShutdownTimeout:    5 * time.Second,
 		HealthCheckInterval:        30 * time.Second,
 		HealthCheckCriticalTimeout: 90 * time.Second,
+		IsPublishing:               true,
```

Comment:
```suggestion
		IsPublishing:               false,
```

`false` is safer - need to turn off extra functionality by default, please